src/unity/Assets/Crest/Shaders/Resources/ShapeCombine.shader

```diff
@@ -44,6 +47,17 @@ Shader "Hidden/Ocean/Simulation/Combine Animated Wave LODs"
 
 			uniform float _HorizDisplace;
 			uniform float _DisplaceClamp;
+			uniform float _CrestTime;
+
+			void Flow(out float2 offsets, out float2 weights)
+			{
+				const float period = 3. * _LD_Params_0.x;
```

Comment:
`OceanNormalMapping.hlsl` also does this (with a different period as well?). So we probably want to have that call this function.

Reply:
```hlsl
	const float half_period = 1;
	const float period = half_period * 2;
	float sample1_offset = fmod(_CrestTime, period);
	float sample1_weight = sample1_offset / half_period;
	if (sample1_weight > 1.0) sample1_weight = 2.0 - sample1_weight;
	float sample2_offset = fmod(_CrestTime + half_period, period);
	float sample2_weight = 1.0 - sample1_weight;
```

Is the code I'm referring to.

Reply:
cool thanks. for the other uses we'll need to decide on the period if we switch to this function, and it would be good to test a similar approach to choosing the period for the normals as these are scaled with lods (whereas foam is not.. currently..). this will take some thinking and testing and i think im going to take the lazy route and tackle that in the future if that sounds acceptible for now.

Reply:
Sounds good 👍